titus-server-master/src/main/java/com/netflix/titus/master/scheduler/DefaultSchedulingService.java

```diff
@@ -531,6 +531,9 @@ private void processTaskSchedulingFailureCallbacks(Map<FailureKind, Map<V3Queuea
     }
 
     private void launchTasks(TaskAssignments taskAssignments) {
+        if (taskAssignments.getCount() <= 0) {
```

Comment:
This isn't entirely a no-op as it sets a gauge

Reply:
I guess it doesn't matter though as it is actually adding a 0 to the AtomicLong which is getting polled.

Reply:
exactly, this mirrors the previous behavior better, before #881